apps/full-site-editing/full-site-editing-plugin/homepage-posts/README.md

```diff
@@ -0,0 +1,30 @@
+# Homepage Posts
+
+This block allows you to list your posts in various layouts and filter them by criteria like category, tag or author.
+
+It originally comes from the [Newspack Blocks collection](https://github.com/automattic/newspack-blocks) and the block is still being developed there.
+
+It was added to this repository in order to make it available to other parts of the FSE plugin, such as Starter Page Templates where this block will be used.
+
+## Structure
+
+```
+index.php — main entry file, registers the block on backend
+editor.js — assets for the block editor
+view.js — assets for the block rendered on frontend
+newspack-homepage-articles/** — source code synced from the Newspack Blocks repository
+```
+
+### Synchronizing the code
```

Comment:
Could we add some notes about development?

-----

### Developmenting / Synchronizing the code

Theoretically, the whole development of this plugin must be exclusively done on the Newspack side. The only purpose of the files not-synced of this plugin is to install the correct infrastructure to integrate/convert the NHA block on/to this Homepage posts block.
Change the code of these files when you need to work on this integration process.

You can see that `newspack-homepage-articles` is being synced...

--------

my two coins. What I want to evidence is the purposes of the non-synced files.